pkg/apis/flow/v1alpha1/jobflow_types.go

```diff
@@ -44,6 +44,12 @@ type Flow struct {
 	Name string `json:"name"`
 	// +optional
 	DependsOn *DependsOn `json:"dependsOn,omitempty"`
+	// Defaults to 1.
```

Comment:
Why not 3 and  be consistent with vc job's default value?

Reply:
Thanks @Monokaix 
Got it
Default setting 3